src/raw/lined/minimize-2.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" width="24" height="24" stroke="currentColor">
```

Comment:
`minimize-2` and `photo-shrink` are very similiar. my vote is to delete `photo-shrink` as it's not semantically correct